### PR TITLE
aws-pricing: add page

### PR DESCRIPTION
**Pull request by piraces**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** 2.7.25

Related: #5653

### CODE REVIEW

File: pages/common/aws-pricing.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# aws pricing
+
+> Centralized and convenient way to programmatically query Amazon Web Services for services, products, and pricing information.
+> More information: <https://docs.aws.amazon.com/cli/latest/reference/pricing/>.
+
+- Get a list of service codes to query later for:
+
+`aws pricing describe-services --region {{us-east-1}}`
+
+- Get a a list of attributes for a given service code:
```
Comment (navarroaxel) [inline pages/common/aws-pricing.md:10]: `get` is more for programming that terminal
```suggestion
- List attributes for a given service code in a specific region:
```

File: pages/common/aws-pricing.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# aws pricing
+
+> Centralized and convenient way to programmatically query Amazon Web Services for services, products, and pricing information.
+> More information: <https://docs.aws.amazon.com/cli/latest/reference/pricing/>.
+
+- Get a list of service codes to query later for:
```
Comment (navarroaxel) [inline pages/common/aws-pricing.md:6]:
```suggestion
- List service codes of a specific region:
```

File: pages/common/aws-pricing.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+`aws pricing describe-services --region {{us-east-1}}`
+
+- Get a a list of attributes for a given service code:
+
+`aws pricing describe-services --service-code {{AmazonEC2}} --region {{us-east-1}}`
+
+- Get pricing information for a service code:
```
Comment (navarroaxel) [inline pages/common/aws-pricing.md:14]:
```suggestion
- Print pricing information for a service code in a specific region:
```

File: pages/common/aws-pricing.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+`aws pricing get-products --service-code {{AmazonEC2}} --region {{us-east-1}}`
+
+- Get a list of attribute values to use while filtering:
+
+`aws pricing get-attribute-values --service-code {{AmazonEC2}} --attribute-name {{instanceType}} --region {{us-east-1}}`
+
+- Get pricing information for a service code and attribute using a filter on the instance type and location:
```
Comment (navarroaxel) [inline pages/common/aws-pricing.md:22]: Which `attribute`? :thinking:
```suggestion
- Print pricing information for a service code and attribute using filters for instance type and location:
```

File: pages/common/aws-pricing.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+`aws pricing get-products --service-code {{AmazonEC2}} --filters {{"Type=TERM_MATCH,Field=instanceType,Value=m5.xlarge"}} {{"Type=TERM_MATCH,Field=location,Value=US East (N. Virginia)"}} --region {{us-east-1}}`
```
Comment (navarroaxel) [inline pages/common/aws-pricing.md:24]: Our standard is quotes outside the token syntax
```suggestion
`aws pricing get-products --service-code {{AmazonEC2}} --filters "{{Type=TERM_MATCH,Field=instanceType,Value=m5.xlarge}}" "{{Type=TERM_MATCH,Field=location,Value=US East (N. Virginia)}}" --region {{us-east-1}}`
```

File: pages/common/aws-pricing.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
+# aws pricing
+
+> Centralized and convenient way to programmatically query Amazon Web Services for services, products, and pricing information.
```
Comment (navarroaxel) [inline pages/common/aws-pricing.md:3]: Perhaps this better
```suggestion
> Query services, products, and pricing information from Amazon Web Services.
```

File: pages/common/aws-pricing.md (hunk 1 above)
Comment (piraces) [reply pages/common/aws-pricing.md]: Totally agree!

File: pages/common/aws-pricing.md (hunk 5 above)
Comment (piraces) [reply pages/common/aws-pricing.md]: Sorry for that! Will consider it for future contributions...

File: pages/common/aws-pricing.md (hunk 6 above)
Comment (piraces) [reply pages/common/aws-pricing.md]: Much better. Mine was the official description from AWS docs.

File: pages/common/aws-pricing.md (hunk 4 above)
Comment (piraces) [reply pages/common/aws-pricing.md]: My fault... It does not make sense, I'm removing this part of the sentence.

Comment (piraces) [reply pages/common/aws-pricing.md]: Done!

File: pages/common/aws-pricing.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+
+- List attributes for a given service code in a specific region:
+
+`aws pricing describe-services --service-code {{AmazonEC2}} --region {{us-east-1}}`
+
+- Print pricing information for a service code in a specific region:
+
+`aws pricing get-products --service-code {{AmazonEC2}} --region {{us-east-1}}`
+
+- Get a list of attribute values to use while filtering:
```
Comment (navarroaxel) [inline pages/common/aws-pricing.md:18]:
```suggestion
- List values for a specific attribute for a service code in a specific region:
```

**Review by navarroaxel — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, after @navarroaxel's suggestion

**Pull request merged**